src/main.js

```diff
@@ -136,6 +157,7 @@ async function run(shouldIncludeVerifiedAccounts) {
     elShouldIncludeVerified.disabled = true;
     nonFollowersList = await getNonFollowersList(shouldIncludeVerifiedAccounts);
     renderResults(nonFollowersList);
+    enableCopyToClipboardButton();
```

Comment:
As mentioned above, can be inlined :)

Reply:
Also, can use `.disabled = false` instead of `.removeAttribute` ?
Just for the sake of uniformity.